CMakeLists.txt

```diff
@@ -587,6 +587,16 @@ if(CIRCT_SLANG_FRONTEND_ENABLED)
   endif()
 endif()
 
+#-------------------------------------------------------------------------------
+# Arcilator JIT
+#-------------------------------------------------------------------------------
+
+if(MLIR_ENABLE_EXECUTION_ENGINE)
```

Comment:
Is there a missing check on the JIR architecture support?  Does the existence of the option imply sufficient dependencies to get something working?

Reply:
If `MLIR_ENABLE_EXECUTION_ENGINE` is enabled, it means MLIR has generated the MLIRExecutionEngine library, so all the needed components must be in place. `MLIR_ENABLE_EXECUTION_ENGINE` is defined by MLIR's main CMake file, which does the necessary target checking.